FirebaseRTC/public/reviewPage.html

```diff
@@ -23,10 +23,10 @@
 
 
     
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/bootstrap.min.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/font-awesome.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/templatemo-art-factory.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/owl-carousel.css">
+    <link rel="stylesheet" type="text/css" href="../../assets/css/bootstrap.min.css">
```

Comment:
In the future, make a habit to use direct terms/names of directories and sub directories. An example of specific file pathway directions can be found here: https://github.com/mellychat/webapp/blob/6ee49c0ae171062c6f67f7218d7094988652a327/public/templates/patientSignUp.html#L18
Make similar file pathway specifications for the other hrefs in the file.

Coach joe says do the thing...with the thing

Reply:
The issue is fixed in the new commit.